ndedhia1210/KiranaShop
# Use typescript for the project

Introducing typescript into the project to reap all the strict typing benefits over plain javascript. Although, we still need to enable the "strict" mode in the tsconfig.json file. This PR just takes an initial step to introduce typescript but in "strict":false mode, so we could at least get some benefits of declaring types, interfaces etc..

Test plan: 
1. Checkout this branch
2. Run `npm install`
3. Run `npm run ts:check` <-- this should display in terminal any type related errors in the code
4. Run `npm start` to verify if the app still works after introduction of typescript
